README.md

```diff
@@ -213,3 +97,7 @@ Read more at https://onfido.com/documentation#webhook-security
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create a new Pull Request
+
+## More documentation
```

Comment:
Not sure we need this section as well as it is a duplication of above.

Reply:
I don't know, I like being really clear in case someone just scrolls to the end or something 😄